File: src/URI/URI_1005.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1005 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double nota1 = sc.nextDouble();
		double nota2 = sc.nextDouble();
		
		double media = (nota1 * 3.5 + nota2 * 7.5) / 11;
		
		System.out.printf("MEDIA = %.5f%n", media);
		
		sc.close();

	}

}

File: kotlin/URI_1005.kt
import java.util.*

const val pesoA = 3.5
const val pesoB = 7.5

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val x = sc.nextDouble()
    val y = sc.nextDouble()

    val media: (Double, Double) -> Double = {a, b -> (a * pesoA +  b * pesoB) / 11.0}
    println(String.format("MEDIA = %.5f", media(x,y)))
}